.github/2.0/neural-search.md

```diff
@@ -1,29 +1,29 @@
 # What is Neural Search?
 
-**Neural Search is deep neural network-powered information retrieval.** In academics, it is often termed as Neural IR. The core idea is to leverage the state-of-the-art deep neural network to build *every* component of a search system. 
+The core idea of neural search is to leverage state-of-the-art deep neural networks to build *every* component of a search system. In short, **neural search is deep neural network-powered information retrieval.** In academia, it's often called *neural IR*. 
 
 ### What can it do?
 
-Thanks to the recent advances in Deep Neural Network, a neural search system can go way beyond simple text search. It enables advanced intelligence on all kinds of unstructured data, such as image, audio, video, PDF, 3D mesh.
+Thanks to recent advances in deep neural networks, a neural search system can go way beyond simple text search. It enables advanced intelligence on all kinds of unstructured data, such as images, audio, video, PDF, 3D mesh, you name it.
```

Comment:
```suggestion
Thanks to recent advances in deep neural networks, a neural search system can go way beyond simple text search. It enables advanced intelligence on all kinds of unstructured data, such as images, audio, video, PDF, 3D mesh. You name it.
```

Reply:
I think those changes are actually not done here but show up only because of the merge against master? Rebasing this PR should help I think?

Reply:
@jacobowitz was right, this PR was messed up. Those changes do not belong to this PR